tests/data/generator/t_protocol/protocol.yaml

```diff
@@ -5,6 +5,7 @@ type: protocol
 description: A protocol for testing purposes.
 license: Apache-2.0
 aea_version: '>=0.9.0, <0.10.0'
+protocol_specification_id: some_author/some_protocol_name:0.1.0
```

Comment:
Probably ok to just call it `specification_id` as it is implicitly clear its the protocol